### PR TITLE
add PyNHD

### PR DESCRIPTION
**Pull request by cheginit**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml"
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged)
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details)
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by cheginit** [comment]
@conda-forge/staged-recipes The checks have passed and it's ready for review, thanks!

File: recipes/pynhd/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  run:
+    - python >=3.6
+    - setuptools
+    - pygeoutils
+    - pygeoogc
+    - networkx
+
+test:
+  imports:
+    - pynhd
```
Comment (marcelotrevisani) [inline recipes/pynhd/meta.yaml:31]: Can we try to add ``pip check`` here please?
```suggestion
    - pynhd
  requires:
    - pip
  commands:
    - pip check
```

File: recipes/pynhd/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+build:
+  noarch: python
+  number: 0
+  script: {{ PYTHON }} -m pip install . --no-deps -vv
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+    - setuptools_scm
```
Comment (marcelotrevisani) [inline recipes/pynhd/meta.yaml:21]:
```suggestion
    - setuptools_scm
    - setuptools  >=41.2
```

**Comment by scopatz** [comment]
Thanks!

**Pull request merged**